integration_tests/test_store_register.py

```diff
@@ -92,14 +92,17 @@ def test_registrations_in_a_row_fail_if_too_fast(self):
         # This test has a potential to fail if working off a slow
         # network.
         self.login()
-        snap_name_1 = 'good-snap{}'.format(uuid.uuid4().int)
-        snap_name_2 = 'test-too-fast{}'.format(uuid.uuid4().int)
 
-        self.register(snap_name_1, wait=False)
+        error = None
+        for idx in range(self.test_store.register_count_limit + 1):
+            snap_name = 'test-too-fast{}-{}'.format(uuid.uuid4().int, idx)
+            try:
+                self.register(snap_name, wait=False)
+            except subprocess.CalledProcessError as exc:
```

Comment:
This doesn't really catch that the 11th registration causes a failure.
Maybe, put self.register in a loop 10 times. Then use assertRaises to catch the 11th failure?